hbase-server/src/main/java/org/apache/hadoop/hbase/master/HMaster.java

```diff
@@ -1911,43 +1912,51 @@ public boolean normalizeRegions() throws IOException {
       return false;
     }
 
-    synchronized (this.normalizer) {
+    if (!normalizationInProgressLock.tryLock()) {
```

Comment:
I wrote the check this way because I prefer, whenever possible, to short-circuit a logic path -- the less context a reader needs to have in their mind, the better. In this case, the inverted path is shorter, so i put it up front, so the reader can see what happens and they that path out of their mind.

Actually, looks like I can short-circuit with a `return` statement, so it can be simplified. Thanks for pointing it out.